OrbitLinuxTracing/PerfEvent.h

```diff
@@ -369,6 +369,23 @@ class TaskRenamePerfEvent : public TracepointPerfEvent {
   }
 };
 
+class GenericTracepointPerfEvent : public PerfEvent {
```

Comment:
Could you move this class either after `DmaFenceSignaledPerfEvent` or before `TracepointPerfEvent`? It splits the classes that inherit from `TracepointPerfEvent`.

Reply:
Of course! Thanks!